themes/openideal_theme/openideal_theme.theme

```diff
@@ -30,3 +31,41 @@ function openideal_theme_form_system_theme_settings_alter(&$form, FormStateInter
     'bg-transparent' => t('Transparent'),
   ];
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function openideal_theme_preprocess_page(&$variables) {
+  $node = \Drupal::routeMatch()->getParameter('node');
+  if ($node instanceof NodeInterface && $node->bundle() == 'idea') {
+    $variables['idea_sidebar'] = TRUE;
+  }
+}
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function openideal_theme_preprocess_field(&$variables) {
+  if (\Drupal::routeMatch()->getRouteName() == 'entity.node.canonical' && $variables['field_name'] == 'field_comments' && $variables['entity_type'] == 'node') {
```

Comment:
I believe we can add template specifically for this field and avoid these conditions.